src/misc.jl

```diff
@@ -7,7 +7,7 @@ Return the logarithm of [multivariate gamma function](https://en.wikipedia.org/w
 """
 function logmvgamma(p::Int, a::Real)
```

Comment:
Test for this, i.e. `src/misc.jl:45`, is currently failing because the numerical accuracy is _slightly_ different than `logbeta`. I'm very confused as to why it started failing now as the constants aren't even being used in `logmvgamma`.

This came up before I made the change to use `logπ` btw; that was an attempt to fix it.